server/db/schemas/test_results.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE test_results(
+    test_id SERIAL PRIMARY KEY,
+    patient_id INT NOT NULL,
+    test_result VARCHAR(255) NOT NULL,
+    type_of_test VARCHAR(255) NOT NULL,
+    Date_of_test TIMESTAMP NOT NULL,
+    street_address VARCHAR(255) NOT NULL,
+    street_address_line_two VARCHAR(255),
+    city VARCHAR(255) NOT NULL,
+    postal_code VARCHAR(255) NOT NULL,
+    province VARCHAR(255) NOT NULL,
```

Comment:
I'd probably just add this as a foreign key to the address table

```suggestion
    address_id INT NOT NULL,
    FOREIGN KEY (address_id) REFERENCES addresses (address_id),
```